### PR TITLE
Moving issuance views to issuer (for SIP-48)

### PR DESCRIPTION
**Pull request by justinjmoses**
Here is a proposal for how to reduce size of Synthetix, by moving issuance-specific views over and keeping.

This is a PR into the SIP-48 branch of #451.

These are the contract size of `develop` (not rates stale)
image

And these are the contract size of this:
image

I'm going to add average gas prices for issuance actions here shortly to compare

### CODE REVIEW

**Comment by justinjmoses** [comment]
Hmmm - I tried a few manual reports
* Synthetix.exchange
     * develop: 470k
     * sip-48: 471k
     * this: 471k
* Synthetix.issue & burn
    * develop: 376k & 206k
    * sip-48: 406k, 204k
    * this: 437k & 234k

File: contracts/Issuer.sol [hunk 1]
```diff
@@ -8,13 +8,17 @@ import "./interfaces/IIssuer.sol";
...
 
-// Inheritance
+// Internal references
 import "./IssuanceEternalStorage.sol";
 import "./interfaces/ISynthetix.sol";
 import "./interfaces/IFeePool.sol";
 import "./interfaces/ISynthetixState.sol";
 import "./interfaces/IExchanger.sol";
 import "./interfaces/IDelegateApprovals.sol";
+import "./IssuanceEternalStorage.sol";
```
Comment (krebernisak) [inline contracts/Issuer.sol:18]: Hi! Just browsing the repo to get a feel for it, and I figured I should make my first comment :)

I noticed a duplicated import here. It is ignored by the compiler, but you should probably remove it.

```suggestion
```

Comment (justinjmoses) [reply contracts/Issuer.sol:18]: Yep good spot, thanks!

**Comment by justinjmoses** [comment]
Latest improvements

image

**Pull request merged**